src/main/java/de/plushnikov/intellij/plugin/processor/clazz/log/CustomLogProcessor.java

```diff
@@ -0,0 +1,155 @@
+package de.plushnikov.intellij.plugin.processor.clazz.log;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.intellij.openapi.util.text.StringUtil;
+import com.intellij.psi.PsiAnnotation;
+import com.intellij.psi.PsiClass;
+import de.plushnikov.intellij.plugin.lombokconfig.ConfigDiscovery;
+import de.plushnikov.intellij.plugin.lombokconfig.ConfigKey;
+import de.plushnikov.intellij.plugin.problem.ProblemBuilder;
+import de.plushnikov.intellij.plugin.util.PsiAnnotationUtil;
+import lombok.CustomLog;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author Adam Juraszek
+ */
+public class CustomLogProcessor extends AbstractLogProcessor {
```

Comment:
Please add some tests/testdata for new functionality.

Reply:
Tests added